pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.25.1"
+pytest = "^6.2.2"
```

Comment:
Looks like you didn't install selenium via poetry?

`poetry add selenium --dev`

Note the `dev` flag will put it in the `dev-dependencies` section below, which is useful for skipping them when setting up the production server.